### PR TITLE
chart download

### PR DESCRIPTION
**Pull request by yivanus**
add download toolbox to charts.
给图形添加下载的功能，下载为png图片

### CODE REVIEW

**Comment by yivanus** [comment]
chartdown

**Comment by cgoldberg** [comment]
why is code from #703 included here?

**Comment by thomas-phillips-nz** [comment]
This looks to be a nicer solution than #693, if it's tidied up.

File: locust/static/chart.js [hunk 1]
```diff
@@ -31,6 +31,16 @@
                     x: 10,
                     y: 10,
                 },
+                toolbox:{
+                	show:true,
+                	feature:{
+                		saveAsImage:{
```
Comment (craig-day) [inline locust/static/chart.js]: does this just give you an image of the graph? If so, it serves a different purpose than #693 which actually gives you all of the data points in a CSV file

Comment (yivanus) [reply locust/static/chart.js]: yes。only the graph。

**Comment by cyberw** [comment]
Hi @yivanus ! Is this still relevant? If so then please resolve the conflicts and we can have a look.

**Pull request closed without merging**